src/app/application.cpp

```diff
@@ -71,7 +72,23 @@
 #include "base/bittorrent/session.h"
 #include "base/bittorrent/torrenthandle.h"
 
-static const char PARAMS_SEPARATOR[] = "|";
+namespace
+{
+#define FILELOGGER_SETTINGS_KEY(name) "FileLogger/" name
+    const QString KEY_FILELOGGER_ENABLED = FILELOGGER_SETTINGS_KEY("Enabled");
```

Comment:
>@glassez : Thank you!

!?!?!

If you guys think it is better, I can leave them out of the namespace...